### DISCUSSION TITLE
JUMP time limit

### DISCUSSION THREAD

**Discussion by mihai-sysbio** [LCSB-BioCore/COBREXA.jl#806] [Q&A] [answered]
I've seen drastically different results from Tulip and GLPK. One likely cause is the default time limits they have. In the case of GLPK, I would like to increase it, so I was looking at [setting a time limit in JuMP](https://jump.dev/JuMP.jl/stable/manual/models/#Set-a-time-limit) or [directly in GLPK](https://github.com/jump-dev/GLPK.jl#use-with-jump), but I'm not sure if this is compatible with the COBREXA setup.

**Comment by exaexa** [comment, marked as answer]
Hi,
yeah this should work. I'll convert this to an issue, we can probably have a quite generic set_time_limt modification.
Thanks for the idea!
